draft-pda-core.md

```diff
@@ -35,21 +26,26 @@ TODO
 
 # Introduction
 
-This document describes an instantiation of Prio, a cryptosystem system designed
-by Henry Corrigan-Gibbs and others [GB17, BGG+19] that allows for
-privacy-preserving computation of statistics over sensitive user data. It works
-by distributing the computation over a set of servers in such a way that, as
-long as at least one server executes the protocol honestly, no input is ever
-seen in the clear.
-
-The goal of this document is to specify a protocol for executing Prio
-computations among a set of servers. We begin in {{overview}} with an overview
-of the protocol and a brief introduction cryptographic techniques underlying
-Prio. In the next section ({{security-requirements}}) we enumerate the security
-goals and non-goals of our protocol. In section {{operational-requirements}} we
-list the use cases our system needs to support, how it is configured, and any
-other operational constraints for the protocol. In {{protocol}} we specify the
-protocol itself.
+This document describes a framework for specifying protocols for
+privacy-preserving data-aggregation. Each protocol is executed by a large set of
+clients and a small set of servers. The servers' goal is to compute some
+aggregate statistic over the clients' inputs without learning the inputs
+themselves. This is made possible by "distributing trust" among the servers in
+such a way that, as long as at least one of them executes the protocol honestly,
+no input is never seen in the clear by any server.
+
+The goal of this document is to specify a protocol for executing private
+aggregation (PA) tasks. We begin in {{overview}} with an overview and brief
+introduction to the underlying cryptographic techniques. In the next section
+({{pa}}) we specify a generic framework for PA protocols. in {{prio}} we
+describe the framework's instantiation with Prio {{BC17}}, and in {{hits}} we
+describe its instantiation with the "heavy hitters" protocol of Boneh et al.
+{{BBCp21}}.
+
+In section {{security-requirements}} we enumerate the security goals and
+non-goals of our protocol. In section {{operational-requirements}} we list the
+use cases our system needs to support, how it is configured, and any other
+operational constraints for the protocol.
```

Comment:
I would rework this and the previous graf. It's not conventional in specifications to provide this kind of "in this section, we do X, then in the next section we do Y". Rather,  we just try to make the specification evolve clearly.

Reply:
Ack, I'll remove the paragraphs for now.